doc/mgr/dashboard.rst

```diff
@@ -136,6 +136,20 @@ of the ``ceph-mgr`` instance, usually the hostname)::
   $ ceph config-key set mgr/dashboard/$name/crt -i dashboard.crt
   $ ceph config-key set mgr/dashboard/$name/key -i dashboard.key
 
+SSL can also be disabled by setting this configuration value:
```

Comment:
Please end that sentence with a double colon, so the command is printed in monospaced font. Currently it looks as follows:

![screenshot-2018-6-21 dashboard plugin ceph documentation](https://user-images.githubusercontent.com/263427/41729658-0f237966-757a-11e8-82df-d368eea35719.png)